app/views/tag/show.html.erb

```diff
@@ -20,7 +20,9 @@
 
       <% end %>
       <% if @wiki %>
-        <p class = "wiki-sub"><%= @wiki.latest.body.split('\n').first %></p>
+        <% 3.times do |i| %>
+          <span class="wiki-sub"><%= @wiki.latest.body.split(' ').slice!((i * 6)..(i * 6) + 5).join(' ') %></span>
```

Comment:
I found a method that actually does this for us: https://apidock.com/rails/ActionView/Helpers/TextHelper/truncate.

It truncates the string and automatically adds `...`. We should use this I think

Reply:
ok will look into it! Thanks for the review :)